genGo.go

```diff
@@ -221,6 +221,16 @@ func (gen *CodeGenerator) GoComplexType(v *ComplexType) {
 			}
 			content += fmt.Sprintf("\t%s\t%s%s\t`xml:\"%s\"`\n", genGoFieldName(element.Name), plural, fieldType, element.Name)
 		}
+
+		if len(v.Base) > 0 {
+			// For valid XSD, `Value` cannot already be used as a separate field
```

Comment:
I'm not sure if this is the best place for this comment. It applies regardless of the language for which we're generating code, but I didn't want to copy it into every code-gen file.